src/main/java/dev/walshy/hardcoreslimefun/utils/Config.java

```diff
@@ -36,7 +36,7 @@ public class Config {
     private String androidMalfunctioned;
 
     public void load(@Nonnull FileConfiguration config) {
-        resetResearchOnDeath = config.getBoolean("on-death.reset-random-research", true);
+        resetResearchOnDeath = getPercent(config,"on-death.resetResearchChance",70);
```

Comment:
Your option key is inconsistent with the one in the config.

Reply:
Sorry, I copied my code onto github and made a mistake.